json-definitions/v3/post/complete/request/index.json

```diff
@@ -0,0 +1,218 @@
+{
+  "title": "Tech Record POST Request Complete Car Schema",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "msUserDetails",
+    "vin",
+    "primaryVrm",
+    "trailerId",
+    "techRecord_reasonForCreation",
+    "techRecord_vehicleType",
+    "techRecord_statusCode",
+    "techRecord_regnDate",
+    "techRecord_manufactureYear",
+    "techRecord_noOfAxles",
+    "techRecord_vehicleSubclass",
+    "techRecord_vehicleConfiguration",
+    "techRecord_euVehicleCategory",
+    "techRecord_applicantDetails",
+    "techRecord_createdAt",
+    "techRecord_createdByName",
+    "techRecord_createdById",
+    "techRecord_lastUpdatedAt",
+    "techRecord_lastUpdatedByName",
+    "techRecord_lastUpdatedById"
+  ],
+  "properties": {
+    "msUserDetails": {
+      "properties": {
+        "msOid": {
+          "type": "string"
+        },
+        "msUser": {
+          "type": "string"
+        }
+      }
+    },
+    "primaryVrm": {
+      "type":
+        "string"
+    },
+    "trailerId": {
+      "type": [
+        "string",
+        null
+      ]
+    },
+    "techRecord_reasonForCreation": {
+      "type": [
+        "string",
+        null
+      ]
+    },
+    "techRecord_vehicleType": {
+      "type": "string",
+      "contains": "car"
+    },
+    "techRecord_statusCode": {
+      "type": "string",
+      "enum": [
+        "current",
+        "provisional",
+        "archived"
+      ]
+    },
+    "techRecord_regnDate": {
+      "type": [
+        "string",
+        null
+      ]
+    },
+    "techRecord_manufactureYear": {
+      "type": [
+        "string",
+        null
+      ]
+    },
+    "techRecord_noOfAxles": {
+      "type": [
+        "integer",
+        null
+      ]
+    },
+    "techRecord_vehicleSubclass": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "enum": [
+          "n",
+          "p",
+          "a",
+          "s",
+          "c",
+          "l",
+          "t",
+          "e",
+          "m",
+          "r",
+          "w"
+        ]
+      }
+    },
+    "techRecord_vehicleConfiguration": {
+      "type": "string"
+    },
+    "techRecord_euVehicleCategory": {
```

Comment:
If you take a look at the `feature/CB2-8067` branch, I have pulled some enums like these to seperate files so they can be used in multiple schemas.